src/bin/opensmt.cc

```diff
@@ -155,7 +154,12 @@ int main( int argc, char * argv[] )
     return 0;
 }
 
-void interpretInteractive(Interpret & interpret, FILE* input) {
+#ifndef ENABLE_LINE_EDITING
+void interpretInteractive(Interpret & interpret) {
+    interpret.interpPipe();
+}
+#else
+void interpretInteractive(Interpret & interpret) {
```

Comment:
Nice! So the `FILE*` parameter was not used at all?
This reminds me that we should try to enable more compiler warnings such as detection of unused parameters and variables :)

Reply:
Yes, looks like it was never implemented.  Maybe not needed.